app/src/main/res/values/strings.xml

```diff
@@ -192,8 +192,8 @@
     <string name="pref_title_connection">Connections</string>
     <string name="summary_paste_certificate">Paste the certificate for dcrd</string>
     <string name="pref_header_about">About</string>
-    <string name="app_version">0.1 alpha build 3</string>
-    <string name="build">Build Date</string>
+    <string name="app_version">Build date {09 Apr 2018}</string>
```

Comment:
Date should not be hard coded. It should dynamically use the current date during the build.